client/app/components/conversation.coffee

```diff
@@ -42,9 +42,6 @@ module.exports = React.createClass
             length = MessageStore.getConversationsLength().get conversationID
             selectedMailboxID ?= Object.keys(message.get('mailboxIDs'))[0]
 
-        displayConvs = AccountStore.hasConversationEnabled selectedMailboxID
```

Comment:
@aenario : Could you explain what weird behaviour could happen? I've made some check : messages are displayed into TrashBox, I can reset it.